### ISSUE TITLE
Storybook fails to restart in Angular tutorial at config section

### ISSUE THREAD

**Issue by jocfico** [chromaui/learnstorybook.com#104]
In the following section of the Angular tutorial:

https://github.com/chromaui/learnstorybook.com/blob/master/content/angular/en/simple-component.md#config

When prompted to restart the Storybook server with `yarn run storybook` the following errors are generated:

```
ERROR in ./src/styles.less (./node_modules/@angular-devkit/build-angular/src/angular-cli-files/plugins/raw-css-loader.js!./node_modules/postcss-loader/src??embedded!./node_modules/less-loader/dist/cjs.js??ref--10-3!./node_modules/style-loader!./node_modules/css-loader/dist/cjs.js!./node_modules/less-loader/dist/cjs.js!./src/styles.less)
Module build failed (from ./node_modules/less-loader/dist/cjs.js):

var content = require("!!../node_modules/css-loader/dist/cjs.js!../node_modules/less-loader/dist/cjs.js!./styles.less");
          ^
Unrecognised input
      in /Users/blah/blah/blah/blah/taskbox/src/styles.less (line 2, column 12)
 @ ./src/styles.less 2:14-356 21:1-42:3 22:19-361
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/config.js ./node_modules/webpack-hot-middleware/client.js?reload=true ./src/styles.less
```

**Comment by AndrewWeb-Dev** [comment]
I was able to get passed this issue by only including the less loader in the webpack.config.js
```

**Comment by AndrewWeb-Dev** [comment]
Exact same issue here

**Comment by domyen** [comment]
Hey @drew-codes @jocfico I cowrote the guide in React but am not that familiar with Angular which is why this issue has languished. Any chance y'all can help us get to the bottom of it and submit a PR? That would help the Storybook Angular community immensely. 🙏

**Comment by davevr** [comment]
Same issue here as well.

**Comment by jonniebigodes** [comment]
@davevr i've been fiddling with this over a couple of days and i think i have a solution. 

Here is the testing approach i've used to triage this issue. 
- Following the introduction guide, created a new angular website with `npx -p @angular/cli ng new taskbox --style less`.
- cd into it
- Installed Storybook as per instructions with `npx -p @storybook/cli sb init`.
- Copied over the less files needed and changed the the contents of the file `src/styles.less` to the following, `@import "./assets/styles/index.less";`
- Issued `yarn test` (the other commands offer the same output) and i'm presented with:
![learn_storybook_less_problem](https://user-images.githubusercontent.com/22988955/66720618-ff271580-edf6-11e9-9a52-bd339c82de05.png)


If you look carefully to the screenshot and it's actually weird that it's trying to import a relative file/font onto a url.
- Tried a couple of things to see if the issue could be solved, until after doing a bit of "detective work" (pardon the bad pun) led me to [this](https://github.com/less/less.js/issues/417), it dates back 2011 and it's still a issue with less. 
- Following the steps provided for the workaround i changed the import to include the `&css` to the end of the import. Transforming the `fontface.less` file contents into:
```css
@import url('https://fonts.googleapis.com/css?family=Nunito+Sans:400,400i,800&css');
```
- Issued `yarn test` and i'm presented with the following:
![learn_storybook_less_solved](https://user-images.githubusercontent.com/22988955/66720659-bde33580-edf7-11e9-924d-e8c62976c112.png)

Tested with the other modes and the output was consistent. No more issue.

I'm going over the angular documentation as i type this and see if it can be improved for the angular user and soon i'm going to submit a pr with it. Which includes this workaround/solution. But for now can you check on your end if the workaround works for your case?

Feel free to provide feedback

**Comment by jonniebigodes** [comment]
@davevr i've been testing and it seems that the problem is fixed with a new version of Storybook, if you can, update the your dependencies and you should not see the issue anymore, you can read more about it [here](https://github.com/storybookjs/storybook/issues/7877).

**Comment by jonniebigodes** [comment]
@domyen and @tmeasday i'm going to close this as this is already solved with #228
